tests/Temporalio.Tests/Worker/WorkflowWorkerTests.cs

```diff
@@ -1227,7 +1227,7 @@ await ExecuteWorkerAsync<MiscHelpersWorkflow>(
                 // Now query after close to check that is replaying worked
                 var isReplayingValues = await Env.Client.GetWorkflowHandle<MiscHelpersWorkflow>(
                     workflowId).QueryAsync(wf => wf.GetEventsForIsReplaying());
-                Assert.Equal(new[] { false, true }, isReplayingValues);
+                Assert.Equal(new[] { false, true, true }, isReplayingValues);
```

Comment:
This is also due to the evict-on-finish change

Reply:
Took me a sec to grok what was happening here. But now I see that on line 1214 above we issue a post-complete query which now causes full replay where it didn't before